### PR TITLE
docs: print out the generated UUID for sending to support

### PR DESCRIPTION
**Pull request by tarzanek**

### CODE REVIEW

File: docs/troubleshooting/report-scylla-problem.rst [hunk 1]
```diff
@@ -184,6 +184,7 @@ The script contents is  as follows:
    sudo perf record --call-graph dwarf -C 0 -F 99 -p $(ps -C scylla -o pid --no-headers) -g --output ./report/perf.data sleep 10
 
    export report_uuid=$(uuidgen)
+   echo $report_uuid
    tar c report | xz > report.tar.xz
    curl --request PUT --upload-file report.tar.xz "scylladb-users-upload.s3.amazonaws.com/$report_uuid/report.tar.xz"
    echo $report_uuid
```
Comment (annastuchlik) [inline docs/troubleshooting/report-scylla-problem.rst:190]: The command you added is repeated on this line. I'm not sure if is useful here or should be removed. Please let me know.

Comment (tarzanek) [reply docs/troubleshooting/report-scylla-problem.rst:190]: @annastuchlik people copy paste(I didn't want to write they are lazy :-D ), so it's just making it easy to spot in the terminal the uuid they will use for upload
so I'd keep it there

**Review by annastuchlik — APPROVED** [review]
LGTM

File: docs/troubleshooting/report-scylla-problem.rst (hunk 1 above)
Comment (tarzanek) [reply docs/troubleshooting/report-scylla-problem.rst:190]: ok, I see your point now

Comment (tarzanek) [reply docs/troubleshooting/report-scylla-problem.rst:190]: there is actually one more bug
s3.amazonaws is not used anymore

Comment (tarzanek) [reply docs/troubleshooting/report-scylla-problem.rst:190]: or is it?
@avikivity do we use s3 aws bucket for uploading stuff still?
or only GCP?

**Pull request closed without merging**